### PR TITLE
EXPOSERS:  Serialization Abstraction Provider - Serialize

### LINKED ISSUE #8: EXPOSERS: Serialization Providers Abstraction Serialize

### PR DESCRIPTION
**Pull request by cjdutoit**
Closes #8

### CODE REVIEW

File: STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Logic.Serialize.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+namespace LHDS.Core.Tests.Unit.Services.Foundations.Addresses
+{
+    public partial class SerializationAbstractionProviderTests
+    {
+        [Fact]
+        public async Task ShouldSerializeAsync()
+        {
+            // given
+            dynamic dynamicPerson = new
```
Comment (hassanhabib) [inline STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Logic.Serialize.cs]: If we don't really care about this object or it's dynamic model then should we call it `somePerson`?

File: STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Logic.Serialize.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace LHDS.Core.Tests.Unit.Services.Foundations.Addresses
```
Comment (glhays) [inline STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Logic.Serialize.cs]: Correct namespace unless intentional

File: STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Exceptions.Serialize.cs [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using STX.Serialization.Providers.Abstractions.Models.Exceptions;
+using STX.Serialization.Providers.Abstractions.Tests.Unit.Models.Exceptions;
+using Xeptions;
+using Xunit;
+
+namespace LHDS.Core.Tests.Unit.Services.Foundations.Addresses
```
Comment (glhays) [inline STX.Serialization.Providers.Abstractions.Tests.Unit/SerializationAbstractionProviderTests.Exceptions.Serialize.cs]: Correct namespace

**Review by glhays — APPROVED** [review]
With hold request.

**Pull request closed without merging**